win/WIN_ColourValueTextbox.h

```diff
@@ -19,12 +19,12 @@ namespace win
 		void redirectLinkedVariable(uint8_t* newVariable);
 		void valueChangedByTextEntry();
 		void valueChangedExternally(); //TODO
-		
-		bool filterNumerical(const char c[]);
-		void updateAndRerender(std::string newString) override;
+
+		static bool filterNumerical(const char c[]);
```

Comment:
Move to a utility class
**SHOULD**

Reply:
created win::utils namespace and moved filterNumerical there